### ISSUE TITLE
[FEATURE REQUEST] Add callback-function after task is done

### ISSUE THREAD

**Issue by Lilihx** [hibiken/asynq#611]
**Is your feature request related to a problem? Please describe.**
I need a callback to notice consumer after the worker task done.
Does asynq have a callback ?

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by bfwbbrj** [comment]
Can you share some background information? What would the use case be like?

**Comment by lee3164** [comment]
> Can you share some background information? What would the use case be like?

Sometimes, we need to schedule task B after task A is done

**Comment by bfwbbrj** [comment]
> > Can you share some background information? What would the use case be like?
> 
> Sometimes, we need to schedule task B after task A is done

In this case, I believe simply calling `asynq.Client.Enqueue` after task A is done would satisfy your needs.

**Comment by lee3164** [comment]
Yes, because this is a simple case. If we have such a feature, we can get many benefits from it. 
1. Our code is clean, and our schedule logic is in the same place, for example: 
```
func schedule(ctx) {
    client.Enqueue(ctx, taskA)
    client.Enqueue(ctx, taskB, After(TaskA))
    client.Enqueue(ctx, taskC, After(TaskB))
}
```
2. In some cases, we need more complicated and fine control. for example:
```
func schedule(ctx) {
    client.Enqueue(ctx, taskA)
    client.Enqueue(ctx, taskB)
    client.Enqueue(ctx, taskC, After(TaskB, TaskB))
}
```
